.github/container/bump.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+## Parse command-line arguments
+
+usage() {
+cat <<EOF
+Usage: $0 [OPTION]...
+  -b, --only-bump-patches        If provided, update patch files and the patchlist in the manifest, but skip bumping refs
+  -i, --input-manifest  PATH     If set, will clean the patch dir. Default is not to clean
```

Comment:
A bit lost. I think, the description is wrong. `--input-manifest` is a path to manifest file to be used to update the refs. What does it have in common with cleaning the patch dir?

Reply:
Good catch. I described "--only-bump-patches" twice. I updated the description